packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -331,6 +353,12 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
 
   Matrix4? _lastTransform;
 
+  /// Whether scrolling up/down on a trackpad or Magic Mouse should map to
```

Comment:
From the style guide, the first paragraph should be a single sentence that briefly describes what the parameter is.